osu.Game.Tournament/Components/TournamentModDisplay.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Linq;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Textures;
+using osu.Game.Rulesets;
+using osu.Game.Rulesets.UI;
+using osu.Game.Tournament.Models;
+using osuTK;
+
+namespace osu.Game.Tournament.Components
+{
+    public class TournamentModDisplay : CompositeDrawable
+    {
+        private readonly string modAcronym;
+
+        [Resolved]
+        private RulesetStore rulesets { get; set; }
+
+        public TournamentModDisplay(string mod)
+        {
+            modAcronym = mod;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(TextureStore textures, LadderInfo ladderInfo)
+        {
+            var texture = textures.Get($"mods/{modAcronym}");
+
+            if (texture != null)
+            {
+                AddInternal(new Sprite
+                {
+                    FillMode = FillMode.Fit,
+                    RelativeSizeAxes = Axes.Both,
+                    Anchor = Anchor.CentreRight,
+                    Origin = Anchor.CentreRight,
+                    Texture = texture
+                });
+            }
+            else
+            {
+                var ruleset = rulesets.GetRuleset(ladderInfo.Ruleset.Value?.ID ?? 0);
+                var modIcon = ruleset?.CreateInstance().GetAllMods().FirstOrDefault(mod => mod.Acronym == modAcronym);
+
+                if (modIcon == null)
+                    return;
+
+                AddInternal(new ModIcon(modIcon)
+                {
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    Scale = new Vector2(0.5f)
+                });
```

Comment:
The mod icon has a tooltip, which is possibly unintended in a tournament client context (might end up hovering on it by accident and blocking out other elements of the screen). Should probably be disabled.